docs/methods.rst

```diff
@@ -104,22 +103,22 @@ look for image updates with all podman-generated unit files that include the aut
    volume: fetchit-volume
    targets:
```

Comment:
Do all of these targets need to become targetConfigs?

Reply:
yes, I'll update that

Reply:
updated - also brought in your commit